### ISSUE TITLE
cDock not working after shutting down and removing power supply

### ISSUE THREAD

**Issue by lojzak01** [w0lfschild/cDock#35]
Hi, I have tried all tips in this topic https://github.com/w0lfschild/cDock/issues/6
CDock is working when I restarted computer but, if I turn macbook off for time wich is longer than 10 minutes and put it power supply off, than cDock stops working (I'vs tried lot versions now 9.5)
This situation is after latest security update od OS X

could you help me?

**Comment by lojzak01** [comment]
So after one and half day of trying to do stable. The problem was in automatic update.
cDock 9.4 and 9.5 (maybe all v9) does not compatible with Mavericks 10.9.5.

Yesterday night I all uninstall, download cDock 8.1 (that is doesn't compatible to) after install 8.1 I have to manually download 8.3. 

Use your own skin.
Turn off automatic update and now it looks fine.

**Comment by lojzak01** [comment]
Now its stop working again....

**Comment by w0lfschild** [comment]
I'm not really sure what you're doing but it sounds like the agent isn't running. Anytime the dock restarts theming needs to be re-applied. The helper does this automatically but you can also manually apply by running:

`osascript -e "tell application \"Dock\" to inject SIMBL into Snow Leopard"`

**Comment by lojzak01** [comment]
v9.5 it this time does not functioning for me, SIMBL seems look OK, I've tried EasySIMBL any difference
Now I have 8.3 

one thing, that look very strange, when I applied 
defaults write com.apple.dock hide-mirror -bool true;killall Dock
The Dock starts whit MIRROR not without as should be
I've tried to type:
defaults write com.apple.dock hide-mirror -bool false;killall Dock
At this moment mirror disappeared, and now it looks transparent.

Thank for your reason and testing I will see tomorrow morning whats the difference.

Transparent Dock disappeared after longer power off.
